index.js

```diff
@@ -33,10 +35,12 @@ function openConfig (cb) {
     return tryOpenConfig(path.join(os.tmpdir(), configfile), cb);
   }
 
-  tryOpenConfig(path.join(userHome, configfile), function (err, fd) {
-    if (err) return tryOpenConfig(path.join(os.tmpdir(), configfile), cb);
-    return cb(null, fd);
-  });
+  fs.mkdir(configPath, function () {
```

Comment:
Wouldn't this need to be `mkdirp`?  By that I mean, is there a chance the multiple directory paths will need to be created?